.githooks/pre-commit

```diff
@@ -1,3 +1,7 @@
-#!/bin/sh
+#!/bin/bash
 
-exec tox
\ No newline at end of file
+if [ -z "${DOCS_ENABLE_HTMLPROOFER}" ]; then
+    export DOCS_ENABLE_HTMLPROOFER=false
+fi
```

Comment:
I am wondering, wouldn't we save a couple of pico-seconds if we skipped the test altogether and just explicitly set `DOCS_ENABLE_HTMLPROOFER` to `false`?

Reply:
Saving picoseconds is irrelevant; what I want to be able to do is *enable* link checking on commit by saying

```bash
export DOCS_ENABLE_HTMLPROOFER=true
git commit
```

And if the test isn't there, I can't do that.